auth_token/utils.py

```diff
@@ -17,8 +18,12 @@ def header_name_to_django(header_name):
     return '_'.join(('HTTP', header_name.replace('-', '_').upper()))
 
 
+def generate_two_factor_code(length):
+    return ''.join([str(random.randint(0, 9)) for _ in range(length)])
+
+
 def login(request, user, expiration=True, auth_slug=None, related_objs=None, backend=None, allowed_cookie=True,
-          allowed_header=True):
+          allowed_header=True, two_factor_login=False):
```

Comment:
Is this needed? Isn't enough to just check `TWO_FACTOR_ENABLED`?

Reply:
I.e. you want to provide signle factor login even if two factor is enabled?

Reply:
I think this is better.